modules/cloud-identity-group/README.md

```diff
@@ -0,0 +1,50 @@
+# Cloud Identity Group Module
+
+This module allows creating a Cloud Identity group and assigning owners, managers and members.
+
+## Usage
+To use this module you must either run terraform as a user that has the Super Admin role in Cloud Identity or [enable domain-wide delegation](https://developers.google.com/admin-sdk/directory/v1/guides/delegation) to the service account used by terraform. If you use a service account, you must also grant that service account the Groups Admin role in Cloud Identity.
+
+Please note that the underlying terraform resources only allow the creation of groups with members that are part of the organization. If you want to create memberships for identities outside your own organization, you have to manually allow members outside your organization in the Cloud Identity admin console.
```

Comment:
Nit: potentially external members can be allowed after setting [allowExternalMembers](https://developers.google.com/admin-sdk/groups-settings/v1/reference/groups#json) - this can't be easily automated with TF yet as the provisioner doesn't support the admin-sdk.

Reply:
I think the terraform resource uses the [Directory API](https://developers.google.com/admin-sdk/directory/v1/guides/manage-groups), which doesn't support the allowExternalMembers option.